README.md

```diff
@@ -38,6 +38,8 @@ Fully customizable media player controls using web components (native custom ele
 #### Results (<a href="https://codepen.io/heff/pen/ZEGdBzN?editors=1000" target="_blank">Try the CodePen example</a>)
 <a href="https://codepen.io/heff/pen/ZEGdBzN?editors=1000" target="_blank"><img width="719" alt="Media Chrome Demo" src="https://user-images.githubusercontent.com/166/78526967-834bcb80-7790-11ea-98a3-a5b355e7a55a.png"></a>
 
+<a href="https://media-chrome.mux.dev/examples/" target="_blank">View more examples</a>
```

Comment:
I actually just re-hunted down this URL yesterday for a discussion with someone lol. I'm also not sure where to put this. Let's keep it here for now but give it a little more attention
* make text larger
* Change text to say something like (?)
  > See all of the repo examples in action.